src/index.js

```diff
@@ -197,7 +197,7 @@ const xrState = (() => {
       return result;
     };
   };
-  const _makeTypedArray = _makeSab(4*1024);
+  const _makeTypedArray = _makeSab(8*1024);
```

Comment:
8*1024 as opposed to 8192 for legibility?

Reply:
We needed this extra memory space because we added more fields to the shared xt state.

Reply:
But yes, it probably wouldn’t matter that this is exactly 8k; it’s just easier to reason about.

Typed arrays also have alignment requirements about, so using non-round values could lead up useless memory in the fragmentation. But that’s just nitpicking for this case.